### PR TITLE
Add webhook transformations

### PR DESCRIPTION
**Pull request by flandrade**
- Add a new optional transformation property to the WebhookDefinition entity.

TODO: 

- [x] Pending release 3.3.2

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,7 +2,14 @@
 All notable changes to this project will be documented in this file.
 This project adheres to [Semantic Versioning](http://semver.org/).
 
+<<<<<<< HEAD
 ## Version [3.3.2] - [2019-27-06]
+=======
```
Comment (Khaledgarbaya) [inline CHANGELOG.md]: Looks like you have an unresolved merge conflict

Comment (flandrade) [reply CHANGELOG.md]: Sorry about it, I forgot to save the file before committing!

**Comment by dkBrazz** [comment]
@Khaledgarbaya @flandrade 
This PR is incorrect
`String` type shouldn't be used for the body in transformation, because the body property can be both String and JSON Object

Actually, Contentful UI creates JSON object, and SDK fails on retrieving such webhooks
It is a blocker

**Comment by flandrade** [comment]
@dkBrazz Thanks for pointing it out. Unfortunately, I no longer contribute to this repository, but please feel free to open a PR to fix it!

**Pull request merged**